swagger2.0/oic.sec.ael.swagger.json

```diff
@@ -17,17 +17,7 @@
 		"/AelResURI": {
 			"get": {
 				"description": "This Resource contains the Auditable Events that have been logged on the Device.",
-				"parameters": [
-					{ 
-						"$ref": "#/parameters/interface" 
-					},
-					{
-						"ref": "#/parameters/events-filtered-by-di"
-					},
-					{
-						"ref": "#/parameters/events-filtered-by-category"
-					}
-				],
+				"parameters": [{"$ref": "#/parameters/interface"}],
```

Comment:
These were put here for a reason.  Should we check with the Sec WG before removing filtering based on di or category?

Reply:
These were not discussed with SecWG as far as I remember, and there are no supporting "shall" statements in the spec.
Would adding them later break back-compat?

Reply:
Since we only do local logging, the "di" query doesn't add a lot of value because every AEE will have the same "di".  Adding "di" queries when we add remote logging would be fine, I think.

I'm not sure if adding the "category" query at a later time will break backward compatibility or not.  If we do add it in the future and if a new Client sends this query to an old Device that doesn't support it, what would happen?  Would a Device today ignore the "category" query and just return all "events" or would it send a 4.xx error?

Reply:
This should be checked with CTWG. I assume that Resource would reply with error, if it can't parse the query parameter.

Reply:
It might make sense to add a test to the generic RETRIEVE test case with some random query parameter (e.g. ?invalid=invalid) just to ensure the current Devices will respond with an error with an unknown query parameter.  Then we can feel safe adding new query parameters in the future without affecting current devices.  We wouldn't want them to blindly ignore unknown parameters.

Reply:
@kmaloor 
Do you know by chance what's the current Lite behaviour regarding unknown query parameters?